relay-event-schema/src/protocol/contexts/profile.rs

```diff
@@ -10,8 +10,12 @@ use crate::protocol::EventId;
 #[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
 pub struct ProfileContext {
     /// The profile ID.
-    #[metastructure(required = "true")]
+    #[metastructure(required = "false")]
     pub profile_id: Annotated<EventId>,
+
+    /// The profiler ID.
+    #[metastructure(required = "false")]
+    pub profiler_id: Annotated<EventId>,
```

Comment:
Would we transfer this `profiler_id` from the profile itself, just like we do for `profile_id`?

Reply:
`profiler_id` wouldn't be transferred. Either it's there or it's not.